pkg/injector/envoy_container.go

```diff
@@ -55,7 +52,6 @@ func getEnvoySidecarContainerSpec(pod *corev1.Pod, cfg configurator.Configurator
 		Args: []string{
 			"--log-level", cfg.GetEnvoyLogLevel(),
 			"--config-path", strings.Join([]string{envoyProxyConfigPath, envoyBootstrapConfigFile}, "/"),
-			"--service-cluster", clusterID,
```

Comment:
We need to keep this here as well. This predates Multicluster - it is needed by Envoy.

Reply:
+1

Reply:
+1